dbt_subprojects/daily_spellbook/models/evms/evms_schema.yml

```diff
@@ -514,6 +514,8 @@ models:
         description:  "Chain where proofs are submitted to"
       - name: settlement
         description:  "Chain that is being settled on"
+      - name: is_on_dune
+        description: "Is the chain on Dune"
```

Comment:
Not sure my description is correct enough.

Reply:
lgtm